src/app/components/ThemeByNetwork/index.tsx

```diff
@@ -14,6 +15,10 @@ export const ThemeByNetwork: FC<{ network: Network; children: React.ReactNode }>
   </ThemeProvider>
 )
 
-export const withDefaultTheme = (node: ReactNode) => (
+export const withMainnetTheme = (node: ReactNode) => (
   <ThemeByNetwork network={Network.mainnet}>{node}</ThemeByNetwork>
 )
+
+export const withDefaultTheme = (node: ReactNode) => (
+  <ThemeByNetwork network={getFixedNetwork() ?? Network.mainnet}>{node}</ThemeByNetwork>
+)
```

Comment:
Should we change theme? Unless with fixed ParaTime only, you can have both mainnet and tesnet?

Reply:
This is also being discussed in another thread, so closing this thread.